src/io/flutter/view/FlutterView.java

```diff
@@ -89,61 +90,71 @@ public void loadState(FlutterViewState state) {
     this.state.copyFrom(state);
   }
 
-  public void initToolWindow(@NotNull ToolWindow toolWindow) {
-    final ContentFactory contentFactory = ContentFactory.SERVICE.getInstance();
+  public void initToolWindow(ToolWindow window) {
+    window.setToHideOnEmptyContent(true);
```

Comment:
Interesting! I didn't know about this method. I wonder how it determines whether the view is empty or not?

Reply:
Empty content means that the contents (list of tabs) is empty. We now remove tabs when each flutter application is stopped matching the behavior of the debugger pane for better or worse.